MercuryFrontend/src/data/LanguageContext.js

```diff
@@ -0,0 +1,4 @@
+import React, { createContext } from 'react';
```

Comment:
No need to import React, here. You can leave just:
import { createContext } from 'react';